src/components/TodoList/TodoList.jsx

```diff
@@ -1,44 +1,96 @@
-import React from 'react';
-import './TodoList.scss';
-
-export const TodoList = () => (
-  <div className="TodoList">
-    <h2>Todos:</h2>
+import React, { useState } from 'react';
+import propTypes from 'prop-types';
+import classnames from 'classnames';
 
-    <div className="TodoList__list-container">
-      <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
-          <label>
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
-          </label>
+import './TodoList.scss';
 
-          <button
-            className="
-              TodoList__user-button
-              TodoList__user-button--selected
-              button
-            "
-            type="button"
-          >
-            User&nbsp;#1
-          </button>
-        </li>
+export const TodoList = ({
+  todos,
+  selectedUserId,
+  selectUser,
+}) => {
+  const [query, setQuery] = useState('');
+  const [type, setType] = useState('all');
 
-        <li className="TodoList__item TodoList__item--checked">
-          <label>
-            <input type="checkbox" checked readOnly />
-            <p>distinctio vitae autem nihil ut molestias quo</p>
-          </label>
+  const visibleTodos = todos
+    .filter(todo => todo.title)
+    .filter(todo => todo.title.includes(query))
+    .filter(todo =>
+      (type === 'completed' ? todo.completed : !todo.completed),
+    );
 
-          <button
-            className="TodoList__user-button button"
-            type="button"
-          >
-            User&nbsp;#2
-          </button>
-        </li>
-      </ul>
+  return (
+    <div className="TodoList">
+      <h2>Todos:</h2>
+      <input
+        type="text"
+        value={query}
+        onChange={event => setQuery(event.target.value)}
+      />
+      <select
+        name="todoState"
+        id="todoState"
+        onClick={event => setType(event.target.value)}
+      >
+        <option
+          value=""
+          defaultValue
+          disabled
+        >
+          Select state
+        </option>
+        <option value="completed">Completed</option>
+        <option value="active">Active</option>
+        <option value="all">All</option>
+      </select>
+      <div className="TodoList__list-container">
+        <ul className="TodoList__list">
+          {visibleTodos.map(todo => (
+            <li
+              key={todo.id}
+              className={classnames({
+                // eslint-disable-next-line quote-props
+                'TodoList__item': true,
```

Comment:
```suggestion
                TodoList__item: true,
```
you may use class names here without quotes